homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -31,10 +31,9 @@ def __init__(self, discovery_info):
         """Initialize the LG speakers."""
         self._host = discovery_info.get("host")
         self._port = discovery_info.get("port")
-        properties = discovery_info.get("properties")
-        self._uuid = properties.get("UUID")
+        self._hostname = discovery_info.get("hostname")
```

Comment:
If you're sure it's always there, use square brackets so the code raises when it isn't:

```suggestion
        self._hostname = discovery_info["hostname"]
```